CHANGELOG.en-US.md

```diff
@@ -16,6 +16,48 @@ tag: vVERSION
 
 ---
 
+## 5.17.0
+
+`2024-05-02`
+
+- 🔥🔥🔥 Ant Design StyleProvider supports configuring `layer` for unified downgrade CSS priority. After the downgrade, the style of antd will always be lower than the default CSS selector priority, so that users can override the style (please be sure to check the browser compatibility of [@layer](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer#browser_compatibility)), check [ full documentation](https://ant.design/docs/react/compatible-style#layer-downgrade). [#48229](https://github.com/ant-design/ant-design/pull/48229)
```

Comment:
```suggestion
- 🔥🔥🔥 StyleProvider supports configuring `layer` for unified downgrade CSS priority. After the downgrade, the style of antd will always be lower than the default CSS selector priority, so that users can override the style (please be sure to check the browser compatibility of [@layer](https://developer.mozilla.org/en-US/docs/Web/CSS/@layer#browser_compatibility)), check [full documentation](https://ant.design/docs/react/compatible-style#layer-downgrade) for usage. [#48229](https://github.com/ant-design/ant-design/pull/48229)
```